django_tgbot/management/commands/tgbotwebhook.py

```diff
@@ -14,19 +14,20 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         bot_username = input('Enter the username of the bot (without @): ').lower()
         dst = os.path.join(settings.BASE_DIR, bot_username)
-        init_file = os.path.join(dst, '__init__.py')
+        init_file = os.path.join(dst, 'credentials.py')
```

Comment:
Better if the variable name is changed to something like `credentials_file` as well since it's not holding the `__init__.py` anymore.

Reply:
Thanks for your guidance. I will resolve this issue ASAP.